### ISSUE TITLE
Issue with Rewrite and Write Paragraph

### ISSUE THREAD

**Issue by danrthompson** [louis030195/obsidian-ava#38]
When I select text and run rewrite, nothing happens. Ditto with write paragraph.

I got the search API and link functionality working, so this does not seem to be an issue with my OpenAI API key or anything like that.

When I look in the console when hitting rewrite, this is what I get:
```
POST https://obsidian-ai-c6txy76x2q-uc.a.run.app/v1/text/create net::ERR_CONNECTION_REFUSED

stream | @ | plugin:ava:13
-- | -- | --
  | s | @ | plugin:ava:556
  | await in s (async) |   |  
  | search | @ | plugin:ava:552
  | (anonymous) | @ | app.js:1
```

When I look in the network tab, the response section says: "Failed to load response data: No data found for resource with given identifier."

Basically the same thing happens with Write Paragraph.

I'd be happy to provide you with any additional information that you need.

**Comment by hotkartoffel** [comment]
Hi thanks will be looking at it tomorrow.

We're experimenting a bit more at the moment and will try to stabilize the soon.

What os are you running?
do you happen to use a vpn?

**Comment by ShravanSunder** [comment]
same issue here with mac os and no vpn

**Comment by hotkartoffel** [comment]
Is fixed now.we're hosting the api on a server so there's no setup required anymore 🎉

No need for any api keys


_(Note you're required to pay for this plugin as of Tuesday.)_

**Comment by louis030195** [comment]
PS: we have a free tier
